maintenance_account/models/account_move.py

```diff
@@ -108,6 +108,8 @@ def _set_equipment_category(self):
                 self._prepare_equipment_category_vals()
             )
             self.equipment_category_id = category.id
+            # Re-compute all equipment_category_id from invoice to prevent dupplicate
+            self.move_id.line_ids._compute_equipment_category_id()
```

Comment:
Doesn't seem a good idea to me, if the user set another category, it will be recomputed, with the default one, ins't it?

Reply:
Added https://github.com/OCA/maintenance/pull/309 to commit history